aoa-jupyter-lab

```diff
@@ -35,20 +35,19 @@ RUN export JAVA_HOME
 RUN pip install --quiet --no-cache-dir --upgrade pip
 
 # Install Teradata Extensions and some other dependencies
-ARG JUPYTERHUB_VERSION=1.4.1
+ARG JUPYTERHUB_VERSION=1.5.0
 RUN pip install --quiet --no-cache-dir \
-    'xgboost==0.90' \
-    'sklearn2pmml==0.60.0' \
-    'keras>=2.2.0' \
-    'shap==0.36.0' \
-    aoa>=$AOA_CLI_VERSION \
-    'tdextensions' \
-    jupyterhub==$JUPYTERHUB_VERSION \
-    'jupyterlab-git>=0.30.1' \
-    'jupyter-resource-usage' \
-    's3cmd' \
-    'tensorflow' \
-    'teradataml'
+    "xgboost==0.90" \
```

Comment:
we can probably remove all the DS libraries from this now as we don't use the default installation?

Reply:
Yep, I'll add those to the conda envs for the kernels.